### PR TITLE
Update chroma.py

### PR DESCRIPTION
**Pull request by alexsatv**
Updated the rules for the rules channel (i hope i did the link and key right for mega... im not sure if its in your config or not)

### CODE REVIEW

File: bot/cogs/chroma.py
Comment (alexsatv) [inline bot/cogs/chroma.py:1]: maybe fix the typos first 🥇

File: bot/cogs/chroma.py [hunk 1]
```diff
@@ -33,6 +33,23 @@
...
 import asyncpg
 
+class infoview(discord.ui.View):
+    def __init__(self):
+        super().__init__(timeout=None)
+        self.value = None
+
+    @discord.ui.button(label="Logos")
+    async def klogos(self, interaction: discord.Interaction, button: discord.ui.Button):
+        logosembed = discord.Embed(title="<a:bun:1098764398962671677> CHroma Logos!", description="˃ Please make sure you watermark the logos!\n˃ Use the watermark on every edit\n˃ Do not share this link with anyone outside the group!", color=0x2b2d31)
```
Comment (alexsatv) [inline bot/cogs/chroma.py]: theres one here

File: bot/cogs/chroma.py [hunk 2]
```diff
@@ -254,5 +271,24 @@ async def claimperks(self, ctx: Context):
         else:
             return
 
+    @commands.command(hidden=True)
+    async def info1(self, ctx):
+        embed = discord.Embed(title="Weclome to Chroma", color=0x2b2d31)
```
Comment (alexsatv) [inline bot/cogs/chroma.py:276]: and here

File: bot/cogs/chroma.py [hunk 3]
```diff
@@ -254,5 +271,24 @@ async def claimperks(self, ctx: Context):
...
+        embed.add_field(name="Group Rules", value="• Must be following [remqsi](https://instagra,.com/remqsi), [wqndqs](https://instagram.com/wqndqs) + [chromagrp](https://instagram.com/chromagrp)"
```
Comment (alexsatv) [inline bot/cogs/chroma.py:277]: and here

**Review by reecerobertz1 — APPROVED** [review]

**Pull request merged**